core/src/main/java/org/elasticsearch/index/query/FQueryFilterBuilder.java

```diff
@@ -105,6 +104,12 @@ public Query toQuery(QueryParseContext parseContext) throws QueryParsingExceptio
     }
 
     @Override
+    public QueryValidationException validate() {
+        // nothing to validate
+        return null;
```

Comment:
thinking about it... we can address it in another PR, but we have to validate the inner query here too :) something we'll have to fix in all the compouned queries